chart/values.yaml

```diff
@@ -64,6 +64,19 @@ yugaware:
     annotations: {}
     labels: {}
     terminationGracePeriodSeconds: 30
+    probes: # HTTP healthcheck probes configuration
+      enabled: true # add or not readinessProbe, livenessProbe and startupProbe to the container
```

Comment:
might make sense to have it enabled: false by default, if users need it they can enable it?

Reply:
Makes sense, will make that change and merge.